### PR TITLE
Pirate Bartender basic functionality

### PR DESCRIPTION
**Pull request by mariajosetamayo**
Hi everyone,
I just finished the basic functionality of Pirate Bartender and would greatly appreciate your feedback!

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -54,12 +54,13 @@ var pirateBartender= Object.create(Bartender.prototype)
...
 }
 
 //<-- State modification functions -->
 
 //<-- function that saves the ingredients preferred by the user in the state object -->
-var chosenIngredients = function(state,currentQuestion){
+var chosenIngredients= function(state,currentQuestion){
 	var currentQuestion= window.currentQuestion
 	var questionsArray= window.arrayQuestions
 	if((currentQuestion===questionsArray[0]) && (state.yesBtn)){
```
Comment (navkelley) [inline js/app.js]: This is a bit repetitive, maybe there is another way you can refactor this to encompass all array positions without having to state each.

File: js/app.js [hunk 2]
```diff
@@ -54,12 +54,13 @@ var pirateBartender= Object.create(Bartender.prototype)
 var state= {
 	yesBtn: false,
```
Comment (navkelley) [inline js/app.js]: Add documentation for what this is for

File: js/app.js [hunk 3]
```diff
@@ -77,17 +78,24 @@ var chosenIngredients = function(state,currentQuestion){
 	}
 }
 
+// <-- Function that identifies if user didn't enter anything and makes state.noPreference true -->
+var noDrink= function(state,ingredientsPreference){
+	if(state.ingredientsPreference.length===0){
+		state["noPreference"]=true
+	}
```
Comment (navkelley) [inline js/app.js]: Does this return anything to the user?

File: js/app.js [hunk 4]
```diff
@@ -106,6 +114,11 @@ var renderDrink= function(element){
 	element.html(drinkHTML)
 }
 
+var renderError= function(element){
```
Comment (navkelley) [inline js/app.js]: stay consistent with comments. While this is self-explanatory the comments you made throughout the document help.

File: js/app.js [hunk 5]
```diff
@@ -120,8 +133,8 @@ $(".yesBtn").on("click", function(event){
 	console.log("drink", pirateDrink)
 	if(index===window.arrayQuestions.length){
 		$(".questionContainer").hide()
-        $(".drinkContainer").show()
-        renderDrink($(".drinkIngredients"))
+       		$(".drinkContainer").show()
```
Comment (navkelley) [inline js/app.js]: indentation should be consistent throughout

File: js/app.js [hunk 6]
```diff
@@ -77,17 +78,24 @@ var chosenIngredients = function(state,currentQuestion){
 	}
 }
 
+// <-- Function that identifies if user didn't enter anything and makes state.noPreference true -->
+var noDrink= function(state,ingredientsPreference){
+	if(state.ingredientsPreference.length===0){
+		state["noPreference"]=true
```
Comment (navkelley) [inline js/app.js]: indentation should be consistent

Comment (mariajosetamayo) [inline js/app.js]: Although I am aware that I sometimes I don't indent consistently, I've been experiencing issues with Github changing my indentation when I push my files. Guess I'll have to re-check my code every time I push to Github.

**Pull request merged**